src/DiskIO/DiskThreads/aiops_win32.cc

```diff
@@ -108,7 +110,7 @@ static Mem::Allocator *squidaio_small_bufs = nullptr; /* 4K */
 static Mem::Allocator *squidaio_tiny_bufs = nullptr; /* 2K */
 static Mem::Allocator *squidaio_micro_bufs = nullptr; /* 128K */
 
-static int request_queue_len = 0;
+static size_t request_queue_len = 0;
```

Comment:
More code changes are needed to make an unsigned type safe (and to preserve type consistency). See return types for functions based on request_queue_len (e.g., squidaio_get_queue_len()) and code decrementing request_queue_len.

Reply:
The code paths touching `request_queue_len` seem exactly the same in aiops.cc and aiops_win32.cc except for the use of `WaitForSingleObject` as a mutex.
What am I missing?

Reply:
Thanks for putting in the time to do history digging.
I'd keep fixing the problems you mention as out of scope for this PR, but mark them down for a followup PR. Among the reasons is that I have a stack of followup PR to allow building on mingw, and I'm concerned about increasing the chance of merge conflicts

Reply:
Understood, but, _again_, if you prefer to duplicate aiops.cc problems in aiops_win32.cc, then please explicitly say so in the PR description. If you want me to do that instead, please let me know.

Reply:
I've updated the PR description. Please feel free to refine it if you feel it's not clear enough

Reply:
Done.